pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>io.swagger</groupId>
-  <artifactId>lf-repository-api-client-java</artifactId>
+  <groupId>com.laserfiche</groupId>
+  <artifactId>com.repository.api.client</artifactId>
```

Comment:
Using dash is the convention. Please revert the change.

Reply:
Also, we probably need to change it for client core lib.

Reply:
yes, the group ID has already been registered and yes i don't believe the artifact ID needs to be registered in sonatype